### DISCUSSION TITLE
Network Settings for windows 11 umm

### DISCUSSION THREAD

**Discussion by damishOnUTM** [utmapp/UTM#3384] [UTM Help (macOS)]
So my windows 11 utm is wanting me to get an update bus it restarted but when I click the "next" button to start updating It just gets stuck at "Just a moment..."

what network config should I use. 

MacBook Air M1 8gb ram 256gb storage 2020

<img width="1440" alt="Skärmavbild 2021-12-12 kl  11 45 18" src="https://user-images.githubusercontent.com/96003802/145709340-9af613cb-a267-474a-b4df-92b7f2fd38e6.png">
<img width="1440" alt="Skärmavbild 2021-12-12 kl  11 45 24" src="https://user-images.githubusercontent.com/96003802/145709358-224208f9-6ce4-43a7-bfc2-ccc5f41e0c96.png">

**Comment by conath** [comment]
Moving to Discussions since not a UTM issue.
